docs/index.md

```diff
@@ -0,0 +1,28 @@
+VFS for Git: Virtualized File System for Git

```

Comment:
(not sure where to place this, so i randomly picked this line.)

Should we have a footnote about dropping support for the mac for external reasons.  in case a reader had already seen the earlier version and was wondering where it went (and why) ?

Reply:
That's a good thing for the FAQ, probably. Let me make a note in the issue to add that (along with a description of how VFS for Git and Scalar fit together in our strategy).